aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -448,11 +457,27 @@
     },
```

Comment:
Should we add `kms:ListAliases` in case our customer was enabling encryption for his DBInstance?